components/home/Banner.tsx

```diff
@@ -70,26 +70,26 @@ function MobileBanner() {
     <Flex
       justify={'end'}
       direction={'column'}
-      w={'24.125rem'}
-      h={'23.25rem'}
+      w={'21rem'}
+      h={'23rem'}
       mt={'24px'}
+      mx={'auto'}
       px={'2rem'}
-      py={'1.5rem'}
+      py={'1rem'}
       bgImage="url('/assets/home/display/mob_money_banner.png')"
-      bgSize={'contain'}
+      bgSize={'cover'}
       rounded={'md'}
     >
       <Text
         color={slate}
         fontFamily={'Domine'}
-        fontSize={'1.5rem'}
+        fontSize={'2xl'}
```

Comment:
Nice! 👌 
BTW, if you are using string, you can remove the brackets. Makes it cleaner
```suggestion
        fontSize='2xl'
```